include/EnvelopeGraph.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * EnvelopeGraph.h - Displays envelope graphs
+ *
+ * Copyright (c) 2004-2009 Tobias Doerffel <tobydox/at/users.sourceforge.net>
+ * Copyright (c) 2024-     Michael Gregorius
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#pragma once
+
+#ifndef LMMS_GUI_ENVELOPE_GRAPH_H
+#define LMMS_GUI_ENVELOPE_GRAPH_H
+
+#include <QWidget>
+
+#include "ModelView.h"
+#include "embed.h"
+
+namespace lmms
+{
+
+class EnvelopeAndLfoParameters;
+
+namespace gui
+{
+
+class EnvelopeGraph : public QWidget, public ModelView
+{
+public:
+	EnvelopeGraph(QWidget* parent);
+
+protected:
+	void modelChanged() override;
+
+	void mousePressEvent(QMouseEvent* me) override;
+	void paintEvent(QPaintEvent* pe) override;
+
+private:
+	void toggleAmountModel();
+
+private:
+	QPixmap m_envGraph = embed::getIconPixmap("envelope_graph");
+
+	EnvelopeAndLfoParameters* m_params;
```

Comment:
```suggestion
	EnvelopeAndLfoParameters* m_params = nullptr;
```
Better to initialize it here I think

Reply:
Applied in another branch. See comments for `EnvelopeGraph` above.